.github/workflows/gh-ci.yaml

```
@@ -23,16 +23,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, ]
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9, "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

According to the schedule we are not py3.8+ https://numpy.org/neps/nep-0029-deprecation_policy.html
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```